core/services/aws_service.rb

```diff
@@ -49,6 +49,12 @@ def delete_key_pair(name)
     @client.delete_key_pair(key_name: name)
   end
 
+  # Get information about instances
+  # @return [Aws::EC2::Types::DescribeInstancesResult] instances information
+  def describe_instances
```

Comment:
Please return the concrete data required by the requesting method. Do not return the AWS types. If you do so, then the clients will depend not only our class API, but on the Amazon library too.